_posts/blog/2017-02-22-team_cut_story.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "A Team Cut Story"
+description:
+modified:
+categories: blog
+author: johannes_theilmann
+excerpt: A short story about how to measure a team cut experiment. We change teams a few times already – this article shows how can we learn which set up was actually more productive and how to perform those experiments as smoothly as possible.
+tags: [Team, Story, Agile]
+image:
+date: 2017-02-22T00:00:00-04:00
+---
+At upday we challenge ourselves to achieve technical excellence and a strong understanding of business needs. Due to external and internal circumstances we have quite a volatile team setup. At least each quarter we change either the team members, the PO or the whole structure of our teams - from horizontal to cross-functional to fluent and back again. However our goal is to have stable teams with a clear business orientation while also enabling team members to grow technically and to support each other (eg. reviews, testing, tech discussions).
+
+It is the challenge for each team member to adapt quickly to new teams, skills and goals. I would like to share how we approach building stronger teams and to have a checklist for myself when a new team is set up:
+
+**As a** Product Owner **I want** to set up teams quickly and efficiently in order to get them up ‘n’ running fast and deliver the highest value.
+
+Acceptance Criteria:
+
+* Project goals are set for the next quarter
+* Team set up is agreed with agile coach / scrum master / team-cut captain
+* <a href="http://theteamcanvas.com/use/" target="blank">Team alignment canvas</a> session defined:
+	* Communication tool (slack, skype, email etc.)
+	*	Planning and progress (jira, trello) – setup corresponding meetings
+	* Define roles, component owner, facilitator, lead
+	* Sign a team agreement
+* Depicting the process the team is improving on a <a href="http://www.storiesonboard.com" target="blank">story map</a>
+	* map projects
+* Calculate the <a href="http://blackswanfarming.com/cost-of-delay-divided-by-duration/" target="blank">CD3</a> (cost of delay/duration) and <a href="http://www.scaledagileframework.com/wsjf/" target="blank">WSJF</a> (weighted short jobs first) with the team.
+* PO-team choose projects based on CD3 and/or WSJF
+* Feedback from stakeholders is collected on a confluence page
+* Epics for projects communicate clearly user/business value and metrics (OKRs) to measure the success of an epic.
+* External team members are invited to groomings to make upcoming changes transparent early on and to final reviews to identify potential follow up stories
+
+We already made several experiments with the team setup without really being able to measure the outcomes. We also realised that not all our needs are covered by the existing experiments. We decided to measure the <a href="https://youtu.be/Avs70dZ3Vlk?t=4m27s" target="blank">success of teams via following KPIs introduced by Martin Fowler </a> (devops state of report 2014):
```

Comment:
"But we realized not all our goals were met anyway."?
put "introduced by..." into the brackets as well